backend/src/main/java/de/qaware/mercury/business/image/color/analysis/graphicsmagick/impl/GMTextOutputParserImpl.java

```diff
@@ -0,0 +1,49 @@
+package de.qaware.mercury.business.image.color.analysis.graphicsmagick.impl;
+
+import de.qaware.mercury.business.image.ImageException;
+import de.qaware.mercury.business.image.color.Color;
+import de.qaware.mercury.business.image.color.ColorMap;
+import de.qaware.mercury.business.image.color.analysis.graphicsmagick.GMTextOutputParser;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+@Service
+@Slf4j
+public class GMTextOutputParserImpl implements GMTextOutputParser {
+
+    // Example of parsed output: "1,0: ( 78,175, 52) #4EAF34"
+    private static final String PIXEL_REGEX = "^(?<x>[0-9]+),(?<y>[0-9]+): \\([ ]*(?<r>[0-9]+),[ ]*(?<g>[0-9]+),[ ]*(?<b>[0-9]+)\\) (?<hex>#[0-9A-F]*)$";
```

Comment:
Wieso diesen String nicht gleich ins `Pattern.compile`?

Reply:
Ich finds generell lesbarer, wenn ein sehr langer String benamst ist, anstatt ihn direkt in nem Aufruf zu verwenden. Im Fall von Pattern.compile macht das vermutlich kaum einen Unterschied, aber bei ner Stringzuweiseung seh ich halt, dass da nichts anderes passiert. Bei nem Aufruf muss ich immer schauen, ob da nach dem String noch n Parameter kommt.

Reply:
Jo, generell stimm ich dir zu, bei Pattern.compile macht das imho keinen Sinn.